src/main/primary_entry/script/page/map/infowindow/InfoWindow.js

```diff
@@ -176,7 +233,7 @@ function _buildLinksDiv(supercharger, showDetails) {
     $.each(linkList, function (index, value) {
         if (value !== null) {
             content += value + "";
-            if (((count++) % 3 === 0) && (index !== linkList.length - 1)) {
+            if (count++ == 3) {
```

Comment:
What is this change doing?

Reply:
It makes the link list of a supercharger popup only two lines long instead of being limited to 3 links to each row.